Automattic/jetpack
# Publicize: Add jetpack_publicize_options

Fixes #565

Still working on testing it; posting for easier discussion.

#### Changes proposed in this Pull Request:
* Syncs the output of a new `jetpack_publicize_options` filter to WP.com to allow locally set options to influence Publicize.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* pbAAop-lT-p2

#### Testing instructions:
* Add this to the Jetpack site:
`add_filter( 'jetpack_publicize_options', function() { return array( 'attach_media' => false ); } );`
* Setup Publicize with Twitter on the Jetpack site.
* Publish. See the tweet using the Twitter card instead of embedding the image.

#### Proposed changelog entry for your changes:
* Publicize: Allow site owners to set a filter to disable embedding media directly into the social media post.

[Edited description as WP.com patch required for this has been committed.]
